spell/rwa-onboarding-checklist.md

```diff
@@ -94,18 +104,20 @@
         * [ ] `hope` (operator, pause proxy)
         * [ ] `mate` (operator, pause proxy as fallback if required)
         * [ ] `kiss` (whitelist for `urn` destination address `who`)
-     * [ ] Filable
-        * [ ] `quitTo`
+     * [ ] Filable (`RwaOutputConduit` and `RwaInputConduit`)
```

Comment:
```suggestion
     * [ ] Filable (`RwaSwapOutputConduit` and `RwaSwapInputConduit{,2}`)
```

Reply:
1. as in https://github.com/makerdao/pe-checklists/pull/12#discussion_r1371306471
2. will adjust to contain `Swap`